docs/RELEASING.md

```diff
@@ -52,4 +54,4 @@ merged into the next PATCH release. New features are always done in Developer pr
 ## Developer preview releases
```

Comment:
@elastic/ingestion-team , I'm not very sure about the Developer preview? Do we really want to **release** it?

Reply:
Right now, we don't have a better way to have ent-search depend on SNAPSHOT versions of connectors code. It would be awesome if we could use a github URL for our `ent-search/Gemfile` to depend on connectors, but we've had issues with that approach and the need to have a valid connectors.yml file.

Reply:
Agree that we need the gem in-between release. What I meant here is do we really need to release it to https://rubygems.org/?

Reply:
🤷 It doesn't cost us anything. But no, we could remove the rubygems publishing from the `make release_*_dev` targets, since ent-search will just vendor the gems

Reply:
Update it to use `make build_service build_utility`, so gems will only be generated in `.gems/` but not pushed to RubyGems.org